helm/csi-vxflexos/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
 appVersion: "2.2.0"
-kubeVersion: ">= 1.21.0 < 1.24.0"
+kubeVersion: ">= 1.21.0 < 1.25.0"
 # If you are using a complex K8s version like "v1.21.3-mirantis-1", use this kubeVersion check instead
 # WARNING: this version of the check will allow the use of alpha and beta versions, which is NOT SUPPORTED
-# kubeVersion: ">= 1.21.0-0 < 1.24.0-0"
+# kubeVersion: ">= 1.21.0-0 < 1.25.0-0"
```

Comment:
If we make it < 1.25, would that imply 1.24.X is supported as well? Should we just keep it <= 1.24?